### PR TITLE
Update hack folder with update_deps files

### PR DESCRIPTION
**Pull request by beemarie**
updating hack folder with update_deps files.

### CODE REVIEW

**Comment by vagababov** [comment]
You probably need to run clean licenses script as well.

**Comment by beemarie** [comment]
> You probably need to run clean licenses script as well.

Is that a script that already exists?  If not, what are you thinking it should do?

**Comment by vagababov** [comment]
They do in others (probably in upstream hack as well).

**Comment by beemarie** [comment]
> They do in others (probably in upstream hack as well).

It looks like update_deps calls update_licenses:
https://github.com/knative/test-infra/blob/master/scripts/library.sh#L569

Were you looking for something else?

**Comment by vagababov** [comment]
yeah, that should remove them... but they are present in your pr?

**Comment by beemarie** [comment]
@vagababov 
Looking at what update_licenses does...
It makes a call to this go-licenses tool:
`  
  run_go_tool github.com/google/go-licenses go-licenses save "${dir}" --save_path="${dst}" --force || \
    { echo "--- FAIL: go-licenses failed to update licenses"; return 1; }
`
iiuc, the go-licenses tool will loop through the required libs & create subdirectories for each of the required license files:
https://github.com/google/go-licenses/blob/master/save.go#L89

What projects have the licenses files removed, so I can take a peek?
/serving looks like it has all the licenses in these sub directories as well:
https://github.com/knative/serving/tree/master/third_party/VENDOR-LICENSE

Same for a couple of the other knative-sandbox projects:
https://github.com/knative-sandbox/net-contour/tree/master/third_party/VENDOR-LICENSE

**Comment by vagababov** [comment]
cc @chizhg @n3wscott 
I am not sure :)

**Comment by n3wscott** [comment]
Til about this repo!

File: hack/update-k8s-deps.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export GO111MODULE=on
+export K8S_VERSION="${1:-v0.18.8}"
+
+K8S_DEPS=(
```
Comment (n3wscott) [inline hack/update-k8s-deps.sh:24]: You could use buoy to update k8s deps for you too, just use the same code that is in use in go_update_deps and change the domain to k8s.io

**Review by n3wscott — COMMENTED** [review]
/lgtm
/approve

File: hack/update-k8s-deps.sh (hunk 1 above)
Comment (beemarie) [reply hack/update-k8s-deps.sh:24]: Thanks @n3wscott, I didn't know about buoy. I'll create an issue to look into using this instead.

Comment (beemarie) [reply hack/update-k8s-deps.sh:24]: https://github.com/knative-sandbox/async-component/issues/24

**Pull request merged**